LICENSE

```diff
@@ -1,34 +1,24 @@
 This project is provided under the terms of the UNLICENSE or
-the MIT license denoted by the following SPDX identifier:
+the BSD license denoted by the following SPDX identifier:
 
-SPDX-License-Identifier: Unlicense OR MIT
+SPDX-License-Identifier: Unlicense OR BSD-3-Clause
 
 You may use the project under the terms of either license.
 
 Both licenses are reproduced below.
 
 ----
-The MIT License (MIT)
```

Comment:
Probably best to retain a title, to make it clear that the text below is the BSD-3 license.

Reply:
Thanks, done!